modules/async-icq/e2e/icq_test.go

```diff
@@ -64,6 +65,14 @@ func TestInterchainQueries(t *testing.T) {
 		UidGid:     "1025:1025",
 	}
 
+	/*
+		relayerImage := ibc.DockerImage{
+			Repository: "icq-relayer",
+			Version:    "local",
+			UidGid:     "1025:1025",
+		}
```

Comment:
Can this use `ghcr.io/cosmos/relayer:main` ? - _We can change in the future when we tag main branch relayer_

Reply:
This is commented out, but we can actually remove it. I believe the default is using that.

Reply:
default looks to be 2.4.1, will be required for the time being